### ISSUE TITLE
Add command to change output quality flags on the fly

### ISSUE THREAD

**Issue by Foocett** [Rickaym/manim-sideview#99]
I was considering just doing this as a pull request because it seems pretty simple, at least on paper, however, I then remembered that I am kinda stupid and gave up after realizing I don't know how VSC extensions work.

Honestly, a advanced user would probably get even more use out of a command to freely change command line arguments as a whole, but as previously stated, I'm kinda stupid, so I would only be editing output quality flags, hence why I suggested that it have its own dedicated command

At the end of the day, I can just go edit the command line args in settings and edit it there but I'm lazy and this seems easier.

**Comment by Rickaym** [comment]
Hey,
How do you imagine the feature? Are you alking about customizing extra command line arguments? This one should be in the settings.

![Uploading image.png…]()

**Comment by Foocett** [comment]
Yeah I've been working with that, however you get some issues with the sideview if you change the quality like that because it exports to a different folder. In my head I imagine a command in the command palate that you can run, then you just get a list of low, medium, high, etc.
I bring up command line arguments because I'm sure more advanced users would get use out of it, but a quality command I think would be a more relevant feature.

**Comment by Segfault1602** [comment]
Can confirm, adding '-qm' to the commandLineArgs config does not work as the extension seems to always look for something in the 1080p60 folder.

**Comment by Rickaym** [comment]
I will look into this issue. Thank you guys for raising it.

**Comment by PeacErtemli** [comment]
Same here, it's definitely a needed feature. Just wanted to raise +1. 👍

**Comment by confusion12345** [comment]
A hacky way to get around this is to add -ql to the command line args, create a symlink from the 480p15 file to the 1080p60 folder. It's gross but I prefer it to disabling preview, using -pql and then watching in vlc. 

When you want your final render in high quality just remember to remove the -ql from the command line args or change it to --qp or -qk for even higher than 1080p.

**Comment by Rickaym** [comment]
FYI, an alternative way to do this is to use a manim configuration file.

**Comment by confusion12345** [comment]
> FYI, an alternative way to do this is to use a manim configuration file. The extension is able to read and change its settings.

Can you advise on what to add to the manim.cfg because when I add quality = low it did not work. 

Thanks

**Comment by Rickaym** [comment]
> > FYI, an alternative way to do this is to use a manim configuration file. The extension is able to read and change its settings.
> 
> Can you advise on what to add to the manim.cfg because when I add quality = low it did not work.
> 
> Thanks

Yes, it's `low_quality`

**Comment by e-dervieux** [comment]
Oh, my apologies, I did not see that this ticket existed before creating these ones:
  - https://github.com/Rickaym/manim-sideview/issues/101
  - https://github.com/Rickaym/manim-sideview/issues/102

FYI available quality options that can be put into the `config.cfg` file are: 'fourk_quality', 'production_quality', 'high_quality', 'medium_quality', 'low_quality', 'example_quality'. All of them are defined [here](https://docs.manim.community/en/stable/_modules/manim/constants.html).

**Comment by Rickaym** [comment]
This should be fixed in v0.4.1, now on the marketplace and Open VSX. The extension now resolves the output path from manim's own logs instead of recomputing it from static config, and executable/environment discovery got a big overhaul alongside it. Please update and reopen if you still hit this.
